### DISCUSSION TITLE
Please tell me why my account has been suspended and why I haven't received any reply for two weeks

### DISCUSSION THREAD

**Discussion by ghost** [community/community#139460] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I first contacted you on Ticket 2989761, but now it has been almost two weeks and you have not replied to me.

Since this is not my registered email address, I reapplied for this Ticket 3002945 correctly, but you still did not reply after a week.

And my friend also reached out to you during this time, but you responded, so I know you're not responding in chronological order.

Anyway, please get back to me as soon as possible. Please.

**Comment by ghost** [comment]
I have no choice but to contact you here. My suspended account is rainlycoris.

**Comment by ebndev** [comment]
Hi @daydreamwarrioryohane,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
